src/test/predict_validation_test.cpp

```diff
@@ -468,19 +469,94 @@ TEST_F(PredictValidation, RequestIncorrectValueCountShapeAuto) {
 }
 
 TEST_F(PredictValidation, RequestWrongPrecision) {
-    auto& input = (*request.mutable_inputs())["Input_FP32_1_3_224_224_NHWC"];
+    auto& input = (*request.mutable_inputs())["Input_FP32_1_224_224_3_NHWC"];
     input.set_dtype(tensorflow::DataType::DT_UINT8);
 
     auto status = instance->mockValidate(&request);
     EXPECT_EQ(status, ovms::StatusCode::INVALID_PRECISION);
 }
 
 TEST_F(PredictValidation, RequestNegativeValueInShape) {
-    auto& input = (*request.mutable_inputs())["Input_FP32_1_3_224_224_NHWC"];
+    auto& input = (*request.mutable_inputs())["Input_FP32_1_224_224_3_NHWC"];
     input.mutable_tensor_shape()->mutable_dim(1)->set_size(-4);
 
     auto status = instance->mockValidate(&request);
     EXPECT_EQ(status, ovms::StatusCode::INVALID_SHAPE);
 }
 
+class PredictValidationArbitraryBatchPosition : public PredictValidation {
+protected:
+    void SetUp() override {
+        PredictValidation::SetUp();
+
+        networkInputs = ovms::tensor_map_t({
```

Comment:
Depending on which MR comes first - this possibly should be servableInputs

Reply:
done